SquareLine/boards/custom_waveshare_7inch/components/ws_7inch/Kconfig

```diff
@@ -0,0 +1,44 @@
+menu "Board Support Package"
+    menu "I2C"
+        config BSP_I2C_NUM
+            int "I2C peripheral index"
+            default 1
+            range 0 1
+            help
+                ESP32S3 has two I2C peripherals, pick the one you want to use.
+
+        config BSP_I2C_FAST_MODE
+            bool "Enable I2C fast mode"
+            default y
+            help
+                I2C has two speed modes: normal (100kHz) and fast (400kHz).
+
+        config BSP_I2C_CLK_SPEED_HZ
+            int
+            default 400000 if BSP_I2C_FAST_MODE
+            default 100000
+    endmenu
```

Comment:
Since this is not a 'real' BSP, could we remove this section and select I2C peripheral in runtime?